src/openrct2/world/LargeScenery.h

```diff
@@ -14,3 +14,8 @@
 #include "TileElement.h"
 
 rct_scenery_entry* get_large_scenery_entry(int32_t entryIndex);
+
+enum
+{
+    LARGE_SCENERY_ELEMENT_FLAGS2_ACCOUNTED = 1 << 0,
```

Comment:
Why an enum?

Reply:
More flags might follow. We use enums for all flag fields.

Reply:
> We use enums for all flag fields.

Not true. I have been using namespaces for ages because it should most of the time force you to write the prefix (`LARGE_SCENERY_ELEMENT_FLAGS2::`) and still allows you to do bitwise ops if you use constants rather than enum class.

Reply:
Well, the tile element stuff still seems to use it pretty consistently, at any rate.